nilearn/_utils/class_inspect.py

```diff
@@ -26,17 +29,6 @@
     "check_set_params",
     "check_transformer_n_iter",
     "check_transformers_unfitted",
-    # Nilearn checks
-    "check_masker_clean_kwargs",
-    "check_masker_fitted",
-    "check_nifti_masker_clean",
-    "check_nifti_masker_detrending",
-    "check_nifti_masker_fit_list_3d",
-    "check_surface_masker_clean",
-    "check_surface_masker_detrending",
-    "check_nifti_masker_fit_with_3d_mask",
-    "check_nifti_masker_fit_with_4d_mask",
-    "check_nifti_masker_fit_with_empty_mask",
```

Comment:
So we don't need to specify these here anymore?

Reply:
nope